packages/components/navbar/src/NavbarAccount/NavbarAccount.tsx

```diff
@@ -8,11 +8,13 @@ import {
   type ExpandProps,
 } from '@contentful/f36-core';
 import { NavbarMenu } from '../NavbarMenu/NavbarMenu';
+import { Avatar } from '../../../avatar/src/Avatar';
```

Comment:
This should point to the package, not the relative path.

Reply:
done